src/main.rs

```diff
@@ -137,6 +137,10 @@ fn main() {
                 .short("t")
                 .help("Whether to wrap each state group change in a transaction")
                 .requires("output_file"),
+        ).arg(
+            Arg::with_name("no_verify")
+                .short("n")
+                .help("Do not double-check that the compression was performed correctly"),
```

Comment:
It would be nice to specify a trade-off here, such as "Saves time at the cost of potentially generating mismatched state."

@erikjohnston I'm not sure how dangerous applying a change could be if this check happened to fail. Whether it is or not would be good to add to the warning as well.